apps/frontend/src/libs/helpers/get-development-routes/get-development-routes.helper.ts

```diff
@@ -0,0 +1,12 @@
+import { type RouteObject } from "react-router-dom";
```

Comment:
I think this is unnecessary, a good approach overall, but we will remove this page as soon as possible, so I believe this helper is not needed. Just leave a todo comment on this item that it should be removed after components usage

Reply:
Done